### ISSUE TITLE
Bug: Pulling external links no longer works

### ISSUE THREAD

**Issue by jesvtb** [chrisgrieser/shimmering-obsidian#168]
### Bug Description

Pressing shift key doesn't pull up all the external links or opens the selected in browser as indicated in documentation. 

### Relevant Screenshot

![2024-03-20 10 47 33](https://github.com/chrisgrieser/shimmering-obsidian/assets/49384847/6ef88259-a4b9-469b-8f73-2b047c9bbc4a)

### To Reproduce

Tried to look in the script folder but tried not to tamper. 

### Debugging Log

```Text
Action does not log anything in the log.
```

### Workflow Configuration

![Screenshot 2024-03-20 at 10 58 07 AM](https://github.com/chrisgrieser/shimmering-obsidian/assets/49384847/d586771f-a5d0-4c42-a9b2-6c720ee017e9)
![Screenshot 2024-03-20 at 10 57 50 AM](https://github.com/chrisgrieser/shimmering-obsidian/assets/49384847/fe812d3b-5e8b-417f-871b-d90d4aaed0d6)
![Screenshot 2024-03-20 at 4 59 21 PM](https://github.com/chrisgrieser/shimmering-obsidian/assets/49384847/3f547acd-e636-48f2-b950-b5fa245e2069)

### Version & System Information

```Text
[10:58:48.678] ERROR: Shimmering Obsidian[Run Script] /Users/jessicaluo/FileServer/4_COMPUTER/42_Application/Alfred/Alfred.alfredpreferences/workflows/user.workflow.A540CA14-AC0C-4A96-8A5B-B1C44F902655/scripts/version_info.js: execution error: Error: Error: The command exited with a non-zero status. (7)

macOS: Sonoma 14.2.1
Alfred:v5.5
Obsidian: Installer: v1.5.8, Current v1.5.11
version of this Alfred workflow：v4.4.1
version of the Metadata Extractor Plugin (v1.1.0) and the Advanced URI Plugin （v1.40.0)
check whether the files allExceptMd.json, tags.json, and metadata.json are in the folder {vault-path}/.obsidian/plugins/metadata-extractor/ YES
if they are, check if they are non-empty NOT EMPTY
does the file {vault-path}/.obsidian/workspace or the file {vault-path}/.obsidian/workspace.json exist? YES
also, I need a screenshot of this screen
```

### Make sure you have done the following

- [X] I have [updated to the latest version](https://github.com/chrisgrieser/shimmering-obsidian/releases/latest) of this workflow.
- [X] I have read the documentation of the feature I am having problems with.
- [X] I am using Alfred 5. (Alfred 4 is not supported by this workflow anymore.)
- [X] I have updated to the latest version of the Metadata-Extractor and the Advanced URI plugin.
- [X] If the `o…` main search does not work, I made sure that I haven't changed any of the default settings of the Metadata-Extractor. Furthermore, I have tried rerunning `osetup` and it still does not work.
- [X] If notes in general can be found, but recently created or renamed notes cannot, I used the `oupdate` command to manually update the metadata.

**Comment by chrisgrieser** [comment]
thx, fixed in version 4.4.2
https://github.com/chrisgrieser/shimmering-obsidian/releases/download/4.4.2/shimmering-obsidian.alfredworkflow

**Comment by jesvtb** [comment]
Thanks! Works like a charm again.
